### ISSUE TITLE
Account display screen width error

### ISSUE THREAD

**Issue by wbt** [MetaMask/metamask-extension#5558]
**Describe the bug**
When an account has a ridiculously high ETH balance, the default MetaMask account display has width issues when displaying the balance.  For example, see the left and right sides of this display:

![still not wide enough in beta](https://user-images.githubusercontent.com/563406/47243709-eab96400-d3c0-11e8-81a2-bb15ce0112ce.PNG)

**To Reproduce**
1. Use Geth to set up a simple private network in which one account has an extremely large balance.
2. Use MetaMask to connect to that private network and install the private key for the rich account.  (Note that the previous bug which prevented Metamask from connecting to a private network in the new UI appears to have been fixed.)
3. View the account.

**Expected behavior**
The text should fit in the display, especially since it's already being abbreviated with scientific notation.  
A very large balance may be elided (e.g. "904...465.124 ETH") similar to how the address is (and you should be able to click on it to copy the full value, as with the address) OR the balance may use fewer significant digits in scientific notation.  In the left side of the screenshot shown, the fiat conversion also doesn't fit on the screen despite scientific notation being used.  
The elided option is preferred as it allows people to more easily track changes in the smaller-value digits where changes are more likely to occur and may be of interest.

**Browser details (please complete the following information):**
 - OS: Windows 10
 - Browser Chrome 69.0.3497.100
 - MetaMask Version 4.16.0
 - New UI

**Comment by wbt** [comment]
Even with scientific notation, the font size is too big for the width of a large balance.

**To Reproduce**
Steps to reproduce the behavior:
1.    Use Geth to set up a simple private network in which one account has an extremely large balance.
2.    Use MetaMask to connect to that private network and install the private key for the rich account. 
3.    View the account:
![Screenshot](https://user-images.githubusercontent.com/563406/48441509-bb94d900-e759-11e8-962a-b9cff073cf7c.png)
or with government currency as primary:
![Screenshot](https://user-images.githubusercontent.com/563406/48441579-e41cd300-e759-11e8-80c5-f7edee6628af.png)

**Expected behavior**
The number before the decimal point and the full currency symbol are visible, not just when hovering over the value.  If there are fewer digits displayed, OK. If the [ellipsis is moved farther right](https://github.com/MetaMask/metamask-extension/pull/5570#issuecomment-431711357), that's OK too.

**Browser details (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome 70.0.3538.77 (latest stable official 64-bit)
 - MetaMask Version 5.0.2

**Comment by alextsg** [comment]
@wbt Thanks for reporting this. Currently working on a pull request that will fix this.

**Comment by wbt** [comment]
Might the issue be reopened, then, as a reminder that there's still something to work on? I just reinstalled with the latest stable release 6.1.0, and the problem seems to be getting worse rather than better.   
![image](https://user-images.githubusercontent.com/563406/53193620-71848480-35df-11e9-840a-f65bf1374519.png)

**Comment by damster7** [comment]
comment peux t on consulter notre solde

**Comment by wbt** [comment]
> comment peux t on consulter notre solde

Translation from French: 
"how can we see our balance"

What I thought might be an answer / possible workaround for that purpose:  Go into Settings, and change Primary Currency between "ETH" and "Fiat" and then look at the smaller text underneath.  That doesn't work either, though:

![image](https://user-images.githubusercontent.com/563406/53259746-de5e5400-369d-11e9-8c93-dd714a60ca2d.png)

If you right-click on the number you wish to see, then choose Inspect, you should be able to see the full value there:
![image](https://user-images.githubusercontent.com/563406/53259820-182f5a80-369e-11e9-9328-21d60397a5ce.png)

**Comment by danfinlay** [comment]
Not sure if this issue is still live, but @rachelcope is redoing our number precision soon, so this is already addressed in another issue. Rachel can you link to that issue here?

**Comment by tmashuang** [comment]
We are actively improving on showing the exact amount values and truncating the width with an ellipsis, designs are in progress.

**Comment by wbt** [comment]
I don't think this should be closed without an improvement.

**Comment by wbt** [comment]
I still don't think this should be closed without an improvement.

**Comment by wbt** [comment]
I don't think this should be closed without an improvement.

**Comment by hjetpoluru** [comment]
I think this could be closed as the large numbers are truncated and attached the screenshots.<img width="958" alt="Image" src="https://github.com/user-attachments/assets/afa0f3ad-98cc-4125-a238-e0be21ba7112" />
<img width="1787" alt="Image" src="https://github.com/user-attachments/assets/0c261fb9-fdb3-4c02-9a21-f2829f7773ac" />

**Comment by wbt** [comment]
Just as a UX point, that kind of truncation showing only the most significant digits without even showing the exponent seems like just about the least useful option, compared to scientific notation or eliding the middle (which would allow a user to see changes in the less-significant end).
